src/components/avatar/theme.css

```diff
@@ -206,6 +222,10 @@
     height: var(--hero-size);
     min-width: calc(var(--hero-size) - var(--overflow-padding-horizontal) * 2);
   }
+
+  .overlay {
+    background-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxMDgiIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAxMDggMjQiPiAgPHBhdGggZmlsbD0iIzM0NEI2MyIgZmlsbC1vcGFjaXR5PSIuODUiIGQ9Ik0xMjUuNjY0NDY0LDEyMCBDMTEyLjQ4MDgzNiwxMzQuNzI4NjI4IDkzLjMyMjg2NzYsMTQzLjk5NzY2MyA3MiwxNDMuOTk3NjYzIEM1MC42NzcxMzI0LDE0My45OTc2NjMgMzEuNTE5MTYzOCwxMzQuNzI4NjI4IDE4LjMzNTUzNTgsMTIwIFoiIHRyYW5zZm9ybT0idHJhbnNsYXRlKC0xOCAtMTIwKSIvPjwvc3ZnPg==);
+  }
```

Comment:
Not sure if we should just dump a base64 string in here 🤔Can't we just use a square background color and position the overlay down and hide the overflow?

Reply:
I agree on that as well, this would give us more flexibility if we need to change the color or opacity of the overlay at some point

Reply:
Agreed, looking into it!